core/config/locales/en.yml

```diff
@@ -51,11 +51,12 @@ en:
         name: Add File
         current: Current File
       search:
-        button_text: Search
+        button_text: Go
```

Comment:
It does - on it's own. However with "Search" you end up with this:
![screen shot 2015-04-27 at 10 19 32 am](https://cloud.githubusercontent.com/assets/397118/7340760/02d84a0e-ecc7-11e4-9b12-0ce46cd61c41.png)

which also looks odd. Happy for a better term than "Go" for sure. 
Or just have the search triggered by Enter and no button.